vitest.config.ts

```diff
@@ -3,6 +3,7 @@ import { defineConfig } from 'vitest/config'
 export default defineConfig({
   test: {
     globals: true,
+    restoreMocks: true,
```

Comment:
Is this still necessary? I understood that we tried this as a solution to the unhandled rejection errors you were seeing, and it didn't resolve them. Should we remove this to clean it up again, or was it still necessary in the end?

Reply:
It's not strictly necessary, but previously we were inconsistently running either `clearAllMocks` or `resetAllMocks` after each test within each file. Instead of repeating this in every test, we now restore mocks by default after each test.

I think this is a sane default and happy to change it if we ever have at least one use case for mocks to persist across tests.